examples/gallery/seismology/wiggle.py

```diff
@@ -0,0 +1,38 @@
+"""
```

Comment:
I don't think `wiggle` is a seismology-specific module. I recommend moving the example to the "Line" category.

Reply:
okay.